lib/exercism/team.rb

```diff
@@ -56,10 +56,6 @@ def defined_with(options, inviter=nil)
     tags = [options[:slug], options[:name], options[:tags]].join(',')
     self.tags = Tag.create_from_text(tags)
 
-    users = User.find_or_create_in_usernames(potential_members(options[:usernames])) if options[:usernames]
-    users = options[:users] if options[:users]
```

Comment:
I couldn't find a place where `options[:users]` were being used, so I dropped it. Thoughts on this?